File: tests/example-frames/animation/bake-anim.cpp
// $dep ufbx cube_animations.fbx
// $ clang++ ufbx.c main.cpp -lm -o example
// $ ./example
#include <stdio.h>
#include <assert.h>
#include "ufbx.h"

// -- EXAMPLE_SOURCE --

int main(int argc, char **argv)
{
    ufbx_scene *scene = ufbx_load_file("cube_animations.fbx", NULL, NULL);
    assert(scene);

    bake_animations(scene);

    return 0;
}

File: tests/example-frames/animation/bake-anim.rs
// $dep cube_animations.fbx
// $ cargo build
// $ cargo run --quiet
use ufbx;

// -- EXAMPLE_SOURCE --

fn main() {
    let scene = ufbx::load_file("cube_animations.fbx", Default::default()).expect("failed to load scene");
    bake_animations(&scene);
}

